docs/general/clients/jellyfin-vue.md

```diff
@@ -0,0 +1,227 @@
+This is a good idea because:
+    * As of now, stable releases don't exist and there have only been prereleases that don't necessarily meet a quality criteria, but
+    [major development milestones](https://jellyfin.org/posts/vue-vue3/).
+    * Only the latest `unstable` image is supported.
+
+After accessing the instance in your browser of choice, you'll be prompted to add a server. You can use your own server or our demo
+instance, located at `https://demo.jellyfin.org/stable`
+
+:::note
+The server address you need to type is relative to the device you're accessing Jellyfin Vue. For instance,
+if your Jellyfin Server is located at 192.168.0.10 and you have deployed Jellyfin Vue to that same server,
+and your client is 192.168.0.20, the address you need to input is `http://192.168.0.10`, not
+`http://127.0.0.1` or `http://localhost`.
+
+You can learn more about how the connection to your server works
+[in Jellyfin's Vue repository privacy disclaimer](https://github.com/jellyfin/jellyfin-vue#privacy-disclaimer-)
+:::
+
+### Using environment variables
+
+This is an example on how environment variables are set in the container by adding the
+following keys to the `jellyfin_vue` definition in [above's](#recommended-using-docker-compose) `docker-compose.yml`:
+
+```yaml
+environment:
+  HISTORY_ROUTER_MODE: 1
+```
+
+See the [complete reference of environment variables available](https://github.com/jellyfin/jellyfin-vue/wiki/Configuration)
+to further customize Jellyfin Vue to your liking at Jellyfin Vue's wiki.
+
+### Using your own webserver
+
+Since Jellyfin Vue is an SPA web application, you can use any webserver to serve it:
+Apache, nginx (the one used in Jellyfin's Vue docker image), Traefik, etc...
+
+If you already have a [reverse proxy](../networking/index.md) set up, do you want to do more complex stuff
+that it's out of the scope of this documentation like serving at a subpath, in another subdomain, etc...,
+you might want to have Jellyfin Vue served by your own webserver instead of the nginx instance shipped with
+the Docker image.
+
+To achieve that, in the `jellyfin_vue` service definition of the `docker-compose.yml` shown [above](#recommended-using-docker-compose):
+* Remove the `ports` key.
+* Add the following keys, **replacing** `_path_` **with the folder** where your webserver expects Jellyfin Vue's assets:
+
+```yaml
+network_mode: none
+volumes:
+  - _𝘱𝘢𝘵𝘩_:/dest
+# This makes the container do nothing and sleep forever,
+# frontend will be copied to _𝘱𝘢𝘵𝘩_ and will be served by your webserver
+# /setup.sh applies 
+entrypoint: /bin/sh -c '/setup.sh && rm -rf /dest/* && cp -r /usr/share/nginx/html/* /dest && sleep infinity'
+
+```
+
+This approach:
+* Keeps the client always updated in a hassle-free way, as explained [above](#recommended-using-docker-compose).
+* In case you use [environment variables](#using-environment-variables),
+they will still be applied.
+
+
+### Replacing Jellyfin Web
```

Comment:
This section should be removed. We should **not** support replacing the web client at all.

Reply:
@nielsvanvelzen I agree to some extent but this is a resource that must be somewhere: this is one of the common questions. I see it in the same way we discourage the use of Raspberry Pi in docs [but there are docs for it](https://jellyfin.org/docs/general/administration/hardware-acceleration/#raspberry-pi-v4l2).

Do you feel that there's something I can change to make the user even more cautious about this? From the disclaimer note or the header of this section (maybe add NOT RECOMMENDED?) instead?

Reply:
Everything we add to our official documentation is in some way endorsed by the Jellyfin team. Replacing the webui with the vue project is cool for people that just use that specific frontend, but it brings two major issues with it:
1) all web wrapper apps break
2) there is no dashboard access

Mainly the first one will be an issue as it will most definitely cause an increase in "client not working" issues because the client can't find the webui on the server.

In the same way we don't support third party plugins that require webui changes and discourage modification of the web files to change the logo/title of your server.

Reply:
I agree with Neils on this one. If we really *wanted* to document this somewhere then maybe somewhere deep in Vue's own developer documentation with lots of nasty warnings would be the appropriate place

Reply:
@felix920506 @nielsvanvelzen Fair enough, that seems like a good middle-ground